File: solutions/zig/pangram/1/pangram.zig
pub fn isPangram(str: []const u8) bool {
    var counts = [_]u8{0} ** 26;
    for (str) |c| {
        if (c >= 'A' and c <= 'Z') {
            counts[c - 'A'] += 1;
        } else if (c >= 'a' and c <= 'z') {
            counts[c - 'a'] += 1;
        }
    }
    for (counts) |c| {
        if (c == 0) {
            return false;
        }
    }
    return true;
}

File: solutions/php/pangram/1/Pangram.php
<?php
declare(strict_types=1);

function isPangram(string $string): bool
{
    $string = strtolower($string);
    $counts = [];
    for ($i = 0; $i < strlen($string); $i++) {
        $char = $string[$i];
        if ($char >= 'a' && $char <= 'z') {
            $counts[$char] = (isset($counts[$char]) ? $counts[$char] : 0) + 1;
        }
    }
    return count($counts) === 26;
}
